frontend/src/pages/AccountSend.vue

```diff
@@ -292,21 +293,60 @@ function useSendForm() {
   }
 
   const isNativeToken = (address: string) => getAddress(address) === NATIVE_TOKEN.value.address;
-  const getMinSendAmount = (tokenAddress: string) => {
+
+  const getNativeTokenMinSendAmountFallback = (chainId: number): number => {
+    switch (chainId) {
+      case 137: return 1.0; // Polygon
+      default: return 0.01; // everything else
+    }
+  }
```

Comment:
I think we should get rid of this method and just key off the native token address (`0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE`) in `getTokenMinSendAmountFallback`. As you can probably tell, I'm not a fan of special-casing the native token if we don't have to since it just makes things clunkier and harder to read/maintain